File: java/src/main/java/com/android/widget/FloatWindow/NeedPermission/OnPermissionListener.java
package com.android.widget.FloatWindow.NeedPermission;

/**
 * Created by xuzhb on 2021/3/8
 * Desc:权限申请结果回调
 */
public interface OnPermissionListener {

    //授权成功
    void onSuccess();

    //授权失败
    void onFailure();

}

File: kotlin/src/main/java/com/android/widget/FloatWindow/NeedPermission/OnPermissionListener.kt
package com.android.widget.FloatWindow.NeedPermission

/**
 * Created by xuzhb on 2021/3/29
 * Desc:权限申请结果回调
 */
interface OnPermissionListener {

    //授权成功
    fun onSuccess()

    //授权失败
    fun onFailure()

}